Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -84,8 +84,10 @@ - (void)commontInit
     [self.payloadView autoPinLeadingToTrailingEdgeOfView:self.avatarView offset:self.avatarHSpacing];
     [self.payloadView autoVCenterInSuperview];
     // Ensure that the cell's contents never overflow the cell bounds.
-    [self.payloadView autoPinEdgeToSuperviewMargin:ALEdgeTop relation:NSLayoutRelationGreaterThanOrEqual];
-    [self.payloadView autoPinEdgeToSuperviewMargin:ALEdgeBottom relation:NSLayoutRelationGreaterThanOrEqual];
+    //
+    // NOTE: It's critical that we pin to the superview top and bottom _edge_ and not _margin_.
```

Comment:
Important

Reply:
I don't understand this change  - Why would we ever want to be closer to the edge than the margin specifies?

This comment doesn't really clarify anything for me - can you use the comment to document *why* not just *what*?